packages/trpc/server/routers/viewer/organizations/createTeams.handler.ts

```diff
@@ -16,31 +16,57 @@ type CreateTeamsOptions = {
 };
 
 export const createTeamsHandler = async ({ ctx, input }: CreateTeamsOptions) => {
-  const { teamNames, orgId } = input;
+  // Get User From context
+  const user = ctx.user;
 
-  const organization = await prisma.team.findFirst({
-    where: { id: orgId },
-    select: { slug: true, metadata: true },
-  });
-  const metadata = teamMetadataSchema.parse(organization?.metadata);
+  if (!user) {
+    throw new NoUserError();
+  }
+
+  const { teamNames, orgId } = input;
 
-  if (!metadata?.requestedSlug && !organization?.slug)
-    throw new TRPCError({ code: "BAD_REQUEST", message: "no_organization_slug" });
+  if (orgId !== user.organizationId) {
```

Comment:
we using orgId from input so we should validate input.orgId matches with user in ctx